apps/trading-e2e/src/support/step_definitions/markets-page.step.ts

```diff
@@ -1,4 +1,5 @@
 import { And, Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+import { forEach } from 'lodash';
```

Comment:
This isn't used?

Reply:
Yeah there is a couple of them just  removing them now :)